lib/monaco-adapter.js

```diff
@@ -59,9 +59,9 @@ var MonacoAdapter = function () {
    */
   function MonacoAdapter(monacoInstance) {
     /** House Keeping */
-    if (monacoInstance !== null && typeof global !== 'undefined' && global.monaco
-      && !monacoInstance instanceof global.monaco) {
 
+    // Make sure this looks like a valid monaco instance.
+    if (monacoInstance !== null && typeof monacoInstance.getModel === 'function') {
```

Comment:
I think this line is throwing the error when the instance is valid, perhaps this one does the trick:
```if (!monacoInstance || typeof monacoInstance.getModel !== 'function'))```

Reply:
Oops, yeah.  My `!== null` example check was not production ready it turns out.  Sorry. 😢

Reply:
I swear I tested it after copying and pasting. However, let me PR again.